tag_hierarchy/commands/filteroptions.cpp

```diff
@@ -0,0 +1,45 @@
+//
+// Created by Petter Moe Kvalvaag on 2019-10-04.
+//
+
+#include "tag_hierarchy/commands/filteroptions.h"
+
+#include "tag_hierarchy/visitors/filteroptionsvisitor.h"
+
+FilterOptions filteroptions(std::string("filteroptions"));
+
+FilterOptions::FilterOptions(std::string name) : Command(name) {}
+
+std::vector<NodeType>
+FilterOptions::ProcessRequest(std::vector<NodeType> &nodes)
```

Comment:
Not covered by tests!

Reply:
True, but we track coverage, are aware, and it is covered in the API application tests so I am ok with it for now. Going forward, extending test data and creating integration tests will be a priority.